### ISSUE TITLE
crashing

### ISSUE THREAD

**Issue by Shironep** [MulverineX/premid-powercord#5]
Hello, for some reason when i try to start the presence it crash the client in a looping (only stop crashing if i disable the plugin)

**Comment by MulverineX** [comment]
In my experience it enters a crash loop if:
a. premid is active on a site before you start discord
b. discord crashes from other means while premid is active

Both of these are relatively avoidable, to fix it when it happens you must kill all discord processes, disable the premid extension, close sites that are causing activity, enable the premid extension, start discord, then continue as normal. I have no idea how to fix this as with Powercord's current setup it is impossible to debug hard crashes like these, even the shell console does not provide enough information. I would recommend removing unstable plugins and if you use inspect element and it crashes to reinstall discord (remove all residual installation files too).

**Comment by Shironep** [comment]
i tried this but still doesn't worked :/ it crash even if i'm not using any website's presence
> kill all discord processes, disable the premid extension, close sites that are causing activity, enable the premid extension, start discord, then continue as normal

**Comment by MulverineX** [comment]
You disabled the extension?

**Comment by Shironep** [comment]
yeah, it worked now, it doesn't crash, but as soon i open an website that has a presence, it crash

1. Not having any website that uses the presences open
2. disable and enable the extension
3. Start Discord and go to the website
Crash

**Comment by MulverineX** [comment]
That's super weird, I've never experienced that behavior. Are you using the latest PreMiD?

**Comment by Shironep** [comment]
Lol, reinstalled the extension and it's working now  (uninstalled and installed from the Chrome Extensions store again), weird, i thought Chrome automatically update the extension

**Comment by MulverineX** [comment]
well if those work with the premid app then its a bug. a pain in the ass, but a bug. Thanks for the report, I'll look into it. You can either not use those presences or use the app for now.

**Comment by Shironep** [comment]
Also, wanted to know, will presence thumbnail support be added? (since it also crash if i try to enable lol, Youtube presence)

**Comment by BlackOfWorld** [comment]
Crashes for me too.
Extension is latest and site is SoundCloud.

**Comment by MulverineX** [comment]
This should be in a separate issue, opening

**Comment by MulverineX** [comment]
uhhh actually no. I should've tested this first:
![image](https://user-images.githubusercontent.com/12068027/147842831-768f1c8e-9ff4-447b-abd0-11d3e04516f9.png)

**Comment by MulverineX** [comment]
Hmmm, thumbnails aren't on by default for the youtube presence, and when I changed my setting in premid to enable them the client crashed 🤔
